### PR TITLE
Non-critical -- [#2055] Remove extraneous create page from project.jsx

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by Pomax** [comment]
This is likely affected by https://github.com/mozilla/webmaker-android/pull/2136, if possible let's land that first, then rebase this, rather than the other way around (it would be a ton of work to fix 2136 back up)

**Comment by thisandagain** [comment]
Looks good. The only regression I'm seeing as that the first page should be automatically selected when the new project loads.

File: www_src/pages/project/project.jsx [hunk 1]
```diff
@@ -380,56 +378,49 @@ var Project = React.createClass({
...
+        var state = {};
+        var pages = this.formatPages(data.pages);
+
+        // Set cartesian coordinates
+        this.cartesian.allCoords = pages.map(el => el.coords);
+
+        state.pages = pages;
+
+        // If no currently selected page...
+        if (!this.state.selectedEl) {
```
Comment (k88hudson) [inline www_src/pages/project/project.jsx]: Does this work for you?

File: www_src/pages/project/project.jsx [hunk 2]
```diff
@@ -78,8 +77,7 @@ var Project = React.createClass({
       var state = window.Android.getMemStorage('state');
       if (typeof state !== 'undefined' && state !== '') {
         state = JSON.parse(state);
-
-        if (state.params && state.params.page === this.state.params.page) {
+        if (state.params && state.params.project === this.state.params.project) {
```
Comment (k88hudson) [inline www_src/pages/project/project.jsx]: Neat bug!

**Comment by thisandagain** [comment]
Looks great! R+

**Pull request merged**